Google.PowerShell/Common/GcloudCmdlet.cs

```diff
@@ -87,9 +88,34 @@ protected string GetFullPath(string filePath)
                 return filePath;
             }
 
-            // For relative files, qualify them based on where the PowerShell session is.
-            string currentDirectory = SessionState.Path.CurrentLocation.Path;
-            return Path.Combine(currentDirectory, filePath);
+            ProviderInfo provider = null;
```

Comment:
Is it possible to rewrite this method so that the scope of both `provider` and `result` are inside the `try` block? As it stands, the function is kind of confusing.

Perhaps:

````
// If the path is already fully-qualified, go with that.
...
// Try to resolve the path using PowerShell (only applicable for FileSystemProvider)
...
// Just default with the input path and hope for the best.
return filePath;
````